docs/en/ingest-management/getting-started.asciidoc

```diff
@@ -113,8 +113,15 @@ manually on the system where the agent is installed.
 
 To use {fleet} to configure {agent}:
 
-. Enable {fleet} in {ingest-manager}. To do so, click the **{fleet}**
-tab and click **Create user and enable {fleet}**.
+. In {ingest-manager}, click **Settings** and change the defaults, if necessary.
+For self-managed installations, set the URLs for {es} and {kib}, then save
```

Comment:
can we add a note about port please, its easy to forget maybe?
...{kib}, including the http port for both, then save your changes.